### DISCUSSION TITLE
Is there a way to update the color of `AM` stats of git files for helm-ls-git?

### DISCUSSION THREAD

**Discussion by zw963** [emacs-helm/helm#2548] [Q&A] [answered]
Following is the screenshot of `helm-browse-project`, i guess it use `helm-ls-git` to show the status of git managed files.

![image](https://user-images.githubusercontent.com/549126/189979238-18b3b716-573e-41d1-8496-8147e7f160c4.png)

But, i found no matter which theme i use, i could not see the `AM` files clearly.

So, i would be nice if i can change that Blue to another color.

thank you.

**Comment by thierryvolpiatto** [comment, marked as answer]
"Billy.Zheng" ***@***.***> writes:

> So, i would be nice if i can change that Blue to another color.

Wherever you are in Helm, you can hit `C-h c` to have the custom menu,
you will find there the face you are looking for.
Here it is `helm-ls-git-added-modified-face`.

-- 
Thierry

**Reply by zw963** [reply]
Yes, thanks!

do you consider change `helm-ls-git-added-modified-face` to others color?  has anyone else been bothered by this?

or,  which theme are you using?

**Reply by zw963** [reply]
I add some more easy read theme screenshot here.

![image](https://user-images.githubusercontent.com/549126/190054470-33b4c9e2-fbed-457e-9eb0-b1255ea69720.png)

![image](https://user-images.githubusercontent.com/549126/190055506-2008eb4e-ee55-4089-ba48-92d6cf8a6565.png)

**Comment by thierryvolpiatto** [comment]
"Billy.Zheng" ***@***.***> writes:

> Yes, thanks!
>
> do you consider change helm-ls-git-added-modified-face to others
> color?

No, but I can if you suggest me a color.

> has anyone else been bothered by this?

No, AM situation is rarely reached.
BTW I added some actions for AM and AD (will merge soon).

> or, which theme are you using?

A old theme never published.


-- 
Thierry

**Reply by zw963** [reply]
> No, but I can if you suggest me a color.

How about this color?

![image](https://user-images.githubusercontent.com/549126/190109170-a5ba60e3-b6e8-43b3-a448-9de54ffe895e.png)

It really nice than old deep blue.

I guess it set by [this](https://github.com/oneKelvinSmith/monokai-emacs/blob/master/monokai-theme.el#L2847-L2848) line, and it should be `#66D9EF`.

BTW: i consider it not same as the color of `??`, right? In fact, i consider those file not managed by git, we can just leave it use `White` color.

**Reply by zw963** [reply]
@thierryvolpiatto , what do you think?

**Comment by thierryvolpiatto** [comment]
My mail of yesterday for some reason doesn't show up here.
I finally used LightSkyBlue instead, do you like it (it is similar).

**Reply by zw963** [reply]
Yes, better than before, thank you very much!

**Comment by thierryvolpiatto** [comment]
"Billy.Zheng" ***@***.***> writes:

>     No, but I can if you suggest me a color.
>    
> How about this color?

Nice.

> image   
>
> It really nice than old deep blue.
>
> I guess it set by this line, and it should be #66D9EF

I used a slighly different one ("LightSkyBlue").

-- 
Thierry
